.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+@basnijholt
+@th3w1zard1
```

Comment:
I added this file here to make sure that PRs do not accidentally get automerged before clicking on the "Approve" button in the review.